src/main/starlark/builtins_bzl/common/java/java_binary_deploy_jar.bzl

```diff
@@ -276,12 +279,19 @@ def make_deploy_jars_rule(implementation):
             ),
             "_cc_toolchain": attr.label(default = "@" + cc_semantics.get_repo() + "//tools/cpp:current_cc_toolchain"),
             "_java_toolchain_type": attr.label(default = semantics.JAVA_TOOLCHAIN_TYPE),
-            "_java_runtime_toolchain_type": attr.label(default = semantics.JAVA_RUNTIME_TOOLCHAIN_TYPE),
```

Comment:
Is it really important, that we don't have a dependency on the runtime type? Or can we keep this as long as we don't request it in the list of toolchains? Not having to do this would keep the rule definition relatively simpler.

In the case that it is necessary to strip the attribute, could you maybe keep the definition in the dict and strip it away using `merge_attrs` from `src/main/starlark/builtins_bzl/common/rule_util.bzl`

Reply:
I think this is just legacy leftover from the toolchain-resolution migration. Native code was using probably readubg the attribute to find the type to lookup in ctx.toolchains. We can drop these attributes now.